packages/shorebird_cli/lib/src/commands/patch_command.dart

```diff
@@ -293,6 +297,16 @@ https://github.com/shorebirdtech/shorebird/issues/472
         return ExitCode.software.code;
       }
     }
+
+    final ReleaseArtifact releaseAabArtifact;
+    try {
+      releaseAabArtifact =
+          await codePushClient.getAabArtifact(releaseId: release.id);
+    } catch (error) {
+      fetchReleaseArtifactProgress.fail('$error');
```

Comment:
Do we want to error? Wouldn't this break existing users? I would think we would just ignore apps which haven't uploaded their app bundles yet for backward compatibility.

Reply:
Fixed.